content/en/docs/reference/command-line-tools-reference/feature-gates.md

```diff
@@ -303,6 +303,8 @@ different Kubernetes components.
 | `TaintNodesByCondition` | `false` | Alpha | 1.8 | 1.11 |
 | `TaintNodesByCondition` | `true` | Beta | 1.12 | 1.16 |
 | `TaintNodesByCondition` | `true` | GA | 1.17 | - |
+| `TokenRequest` | `true` | GA | 1.20 | - |
```

Comment:
Do NOT remove the history.
Line 155-158 should be moved here.
The goal is to let users see the evolution history of the feature, not just its current state.

Reply:
done. the history was not removed but kept in the last section.